packages/lesswrong/components/sequences/SequencesNavigationLink.tsx

```diff
@@ -51,7 +51,7 @@ const SequencesNavigationLink = ({ post, direction, classes }: {
       </Link>
     )
     if (post.title) {
-      return <Tooltip title={post.title}>{button}</Tooltip>
+      return <Tooltip title={post.title} placement="top">{button}</Tooltip>
```

Comment:
Guessing this is used by the EA forum, but not sure

Reply:
I think this is an improvement for both forums (although the new LW design gets more mileage out of it). Previously the tooltip overlapped the post title.

Reply:
Fairly confident EAFolk will (slightly) prefer this. @s-cheng you can imagine the placement of the arrows here:

https://forum.effectivealtruism.org/s/gBjPorwZHRArNSQ5w/p/yjm5CW9JdwBTFZB2B

Reply:
Seems good, thanks!